app/models/concerns/push_updates_for_subscription.rb

```diff
@@ -39,4 +29,26 @@ def app_name
     end
   end
 
+  def data_to_send
+    data = { category: 'message',
+             message: message_body,
+             is_private: message.is_private,
+             item_id: item_id,
+             author_id: message.sender_id,
+             message_id: message.id }
+    data[identity] = message.related_object.id
+    data
+  end
+
+  def message_body
+    message.body.truncate(150, separator: ' ')
+  end
+
+  def item_id
```

Comment:
I think we can start to remove this from the front-ends now that we will start sending `messageable_id` and `messageable_type`. Perhaps comment the method to say it will be deprecated soon as we can't remove it until the apps are fixed